### PR TITLE
Update #150 add indicators for new internet tables

### PR DESCRIPTION
**Pull request by maggiesu0725**

### CODE REVIEW

**Comment by maggiesu0725** [comment]
@lhendey ready for review

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 1]
```diff
@@ -3935,6 +3964,33 @@
 		mNumOthercomputeronly_&_years.="Total number of households that has other compueters with no other type of computing devices, MOE, &_years_dash  "
 		mNumNocomputer_&_years.="Total number of households that has no computers,MOE,&_years_dash "
 		 
+
+        Numdialup_&_years. = "Total number of households that have a dial up with no other type of Internet subscription,&_years_dash "
+		Numbroadbandall_&_years. = "Total number of households that have broadband subsctiption of any type,&_years_dash "
+		Numcellular_&_years. = "Total number of households that have a Cellualr data plan,&_years_dash "
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas]: misspelling of cellular

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 2]
```diff
@@ -3935,6 +3964,33 @@
...
+		Numcellularonly_&_years. = "Total number of households that have a Cellular data plan with no other type of Internet subscription,&_years_dash "
+		Numbroadband_&_years. = "Total number of households that have broadband subscription such as cable fiber optics or DSL,&_years_dash "
+		Numbroadbandonly_&_years. = "Total number of households that have broadband subscription such as cable fiber optics or DSL wiht no other types of Internet subscription,&_years_dash "
+		Numsatellite_&_years. = "Total number of households that have Satelllite Internet service,&_years_dash "
```
Comment (lhendey) [inline Macros/ACS_summary_geo_source_tr_vars.sas]: misspelling of satellite (multiple)

**Comment by maggiesu0725** [comment]
Thanks, I think you can just add up the others to get the Num1orMoreComp based on the table shell. But I added it to the program. Please review. @lhendey

File: Macros/ACS_summary_geo_source_tr_vars.sas [hunk 3]
```diff
@@ -3820,6 +3821,34 @@
 		mNumOthercomputeronly_&_years.= %moe_sum (var= B28001m10);
 		mNumNocomputer_&_years.= %moe_sum (var= B28001m11);
```
[the remark this answers is not in the document]
Comment (lhendey) [reply Macros/ACS_summary_geo_source_tr_vars.sas]: @maggiesu0725  i looked at a table with data in factfinder and it doesn't appear that's true

**Comment by maggiesu0725** [comment]
Sorry about that, revisions updated. @lhendey should I reverse changes to the sf revisions?

**Comment by lhendey** [comment]
@maggiesu0725 yes reverse the changes to sf.

**Pull request merged**